build_sysext

```diff
@@ -0,0 +1,108 @@
+sudo build_sysext --board=arm64-usr command-extras package_1 package_2 package_3
+
+"
+
+show_help_if_requested "$@"
+
+# Parse command line
+FLAGS "$@" || exit 1
+
+eval set -- "${FLAGS_ARGV:-prod}"
+
+. "${BUILD_LIBRARY_DIR}/release_util.sh" || exit 1
+
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/board_options.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/dev_container_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/test_image_content.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/torcx_manifest.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/cros_vm_constants.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/oem_sysext_util.sh" || exit 1
+
+set -euo pipefail
+
+cleanup() {
+  umount "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" || true
+  rm -rf "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" || true
+}
+
+export SYSEXTNAME="${1-}"
+
+if [ -z "$SYSEXTNAME" ]; then
+    echo "Error: No sysext name passed"
+    show_help_if_requested -h
+fi
+
+# Set up trap to execute cleanup() on script exit
+trap cleanup EXIT
+
+ARCH=$(_get_sysext_arch "${FLAGS_board}") 
+
+source <(grep -P '^(VERSION|VERSION_ID)=' /etc/os-release)
+
+cleanup
+
+mkdir "${BUILD_DIR}/fs-root"
+mount -rt squashfs -o loop,nodev "${BUILD_DIR}/flatcar_production_image_sysext.squashfs" "${BUILD_DIR}/fs-root"
+mkdir "${BUILD_DIR}/install-root"
+mkdir "${BUILD_DIR}/workdir"
+mount -t overlay overlay -o lowerdir="${BUILD_DIR}/fs-root",upperdir="${BUILD_DIR}/install-root",workdir="${BUILD_DIR}/workdir" "${BUILD_DIR}/install-root"
+VERSION_BOARD=$(grep "^VERSION=" ${BUILD_DIR}/fs-root/usr/lib/os-release | cut -d = -f 2-)
+if [ "$VERSION_BOARD" != "$VERSION" ]; then
+  echo "$VERSION_BOARD"
+  echo "$VERSION"
+  echo "Version mismatch between board flatcar release and SDK container flatcar release"
+  exit 1
+fi
+shift
+  
+for package in "$@"; do
+  echo "Installing package into sysext image: $package"
+  ROOT="${BUILD_DIR}/install-root" PORTAGE_CONFIGROOT="/build/${FLAGS_board}" FEATURES="-ebuild-locks" emerge --root-deps=rdeps --usepkgonly --verbose "${package}"
+done
+
+# Unmount in order to get rid of the overlay
+umount "${BUILD_DIR}/install-root"
+umount "${BUILD_DIR}/fs-root"
+rm -rf "${BUILD_DIR}/install-root/dev" "${BUILD_DIR}/install-root/etc" "${BUILD_DIR}/install-root/var" "${BUILD_DIR}/install-root/opt"
+mkdir -p "${BUILD_DIR}/install-root/usr/lib/extension-release.d"
+printf "ID=flatcar\nVERSION_ID=${VERSION_ID}\nARCHITECTURE=${ARCH}\n" >"${BUILD_DIR}/install-root/usr/lib/extension-release.d/extension-release.${SYSEXTNAME}"
+mksquashfs "${BUILD_DIR}/install-root" "${BUILD_DIR}/${SYSEXTNAME}.raw" -all-root
+rm -rf "${BUILD_DIR}/{fs-root, install-root, workdir}"
+upload_image "${BUILD_DIR}/${SYSEXTNAME}.raw"
```

Comment:
```suggestion
```
Let's remove it for now because it won't run as `build_library/release_util.sh` would have to be sourced before flag parsing and anyway we can't test it in this PR.